lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -50,6 +50,7 @@ function divideByGCD(fraction) {
     '/', [intermediateNumerator, intermediateDenominator]);
   const status = Node.Status.nodeChanged(
     ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  status.newNode = Node.Status.resetChangeGroups(status.newNode);
```

Comment:
you can make just the 3s coloured yellow by setting both the 3-node changegroups to 1 (by default the whole thing gets a changegroup, so you'll want to turn off that default in ...arguments to NodeStatus I think?)

Also shouldn't there be another substep for cancelling the 3s?